src/Tools/MicroBuild/run-pdbgit.ps1

```diff
@@ -0,0 +1,84 @@
+<#
+
+    Run PDB Git on our PDB files to enable proper source server support
+
+#>
+Param(
+    [string]$sourcePath = $null,
+    [string]$binariesPath = $null
+)
+
+set-strictmode -version 2.0
+$ErrorActionPreference="Stop"
+
+function Get-PackagesPath {
+    $packagesPath = $env:NUGET_PACKAGES
+    if ($packagesPath -eq $null) {
+        $packagesPath = join-path $env:UserProfile ".nuget\packages\"
+    }
+
+    return $packagesPath
+}
+
+try
+{
+    [xml]$deps = get-content (join-path $sourcePath "build\Targets\Dependencies.props")
+    $pdbGitVersion = $deps.Project.PropertyGroup.PdbGitVersion
+    $packagesPath = Get-PackagesPath
+    $pdbGit = join-path $packagesPath "pdbgit\$pdbGitVersion\tools\PdbGit.exe"
+
+    # Linking our PDBs to Git is really only interesting for PE / PDB that are 
+    # being shipped to customers.  For local developement the original PDB 
```

Comment:
Wow. 15 minutes for 60 PDBs is awful. When I wrote (some of) the PdbGit code, I remember that there was non-performant code I was writing that could be improved if necessary (the whole "optimize after you measure" philosophy). So given how *bad* it is, can you please send an ETL trace of the tool running (perhaps the command line invocation on just one of your larger PDBs)? I bet we can fix it significantly (perhaps an order of magnitude).